### PR TITLE
feat: skip all listening excercises button

### PR DESCRIPTION
**Pull request by DavidLuterancik**
This PR resolves the issue #733

Added a 'Can't listen now' button to skip all the remaining listening exercises

### CODE REVIEW

**Comment by kantord** [comment]
Hi! Sorry for the slow reply, I'm sick and don't have much energy :sneezing_face: :face_with_thermometer: 

The PR looks very nice!

Can you please extend this file to describe this feature? https://github.com/kantord/LibreLingo/blob/master/workspaces/web/cypress/integration/listening.feature you can run `yarn web cy:open` to verify that it runs correctly.

Also, please resolve the conflicts

**Comment by DavidLuterancik** [comment]
Sorry for being slower with the replies too, I've been busier lately. 

I've extended the file as mentioned and resolved the conflicts.

File: workspaces/web/cypress/integration/listening.feature [hunk 1]
```diff
@@ -59,11 +60,21 @@ Feature: Listening exercise
     And I click "Submit"
     Then I don't see a "Skip" button
     And I click "Continue"
-    Then I see a panel with only a skip and a finish early button
+    Then I see a "Skip" button
+    And I see a "Can't listen now" button
+    And I see a "Finish early" button
 
-  Scenario: Going to the next challenge with keyboard only
+  Scenario: Skipping a listening excercise
```
Comment (kantord) [inline workspaces/web/cypress/integration/listening.feature:67]: Why has this one been renamed?

Comment (DavidLuterancik) [reply workspaces/web/cypress/integration/listening.feature]: I must've changed it by accident, fixed it with the newest commit

Comment (kantord) [reply workspaces/web/cypress/integration/listening.feature]: amazing, thank you!

**Comment by kantord** [comment]
:100: 
Thank you, @DavidLuterancik

**Pull request merged**